modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -392,9 +394,14 @@ export default function ObjectFieldFormBase({
 		};
 
 		makeFetch();
+		setReload(true);
```

Comment:
@Artartur, As for now we need to update this state only for the Picklist case, you can move this `setReload(true);` for the `getObjectFieldSettingsByBusinessType` function inside the Picklist and MultiselectPicklist condition:

```tsx
...

if (businessType === 'Picklist' || businessType === 'MultiselectPicklist') {
    setReload(true);
    
    updateListTypeDefinitions(setListTypeDefinitions);
}

...
```

Reply:
Done!